tests/conftest.py

```diff
@@ -126,21 +126,19 @@ def test_with_any_microvm(test_microvm_any):
 METRICS = get_metrics_logger()
 
 
-def pytest_configure(config):
-    """Pytest hook - initialization"""
-    config.addinivalue_line("markers", "nonci: mark test as nonci.")
-
-
 def pytest_addoption(parser):
     """Pytest hook. Add command line options."""
     parser.addoption("--nonci", action="store_true", help="run tests marked with nonci")
+    parser.addoption(
+        "--optional", action="store_true", help="run tests marked as optional"
```

Comment:
super nit: Shall we rename this to something like "--no-block-pr", so that we do not get confused with the typical connotations of an `--optional` cli flag?